### PR TITLE
Feature/login using facebook

### PR DESCRIPTION
**Pull request by tungbinhdinh89**
setup login page and interaction

### CODE REVIEW

**Comment by tungbinhdinh89** [comment]
[https://youtu.be/E3O94KcO3ak](url)

File: src/megax/MegaApp/Vite/src/pages/Login/LoginPage.tsx [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+    handleSubmit,
+    formState: { errors },
+  } = useForm();
+
+  return (
+    <Stack spacing={2} sx={{ width: "100%" }}>
+      <div className="w-full pb-8">
+        <form
+          onSubmit={handleSubmit(values => {
+            console.log(values);
```
Comment (oclockvn) [inline src/megax/MegaApp/Vite/src/pages/Login/LoginPage.tsx:56]: remove console log

File: src/megax/MegaApp/Vite/src/pages/Login/LoginPage.tsx [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+            values.password === defaultValue.passworld
+              ? (setIsSuccess(true), setIsFailed(false))
+              : (setIsFailed(true), setIsSuccess(false));
+          })}
+          className="bg-white shadow-md rounded px-8 pt-6 pb-[80px] mb-4"
+        >
+          <div className=" mb-4">
+            <TextField
+              {...register("username", {
+                required: "Thi is required.",
```
Comment (oclockvn) [inline src/megax/MegaApp/Vite/src/pages/Login/LoginPage.tsx:67]:
```suggestion
                required: "Username is required.",
```

File: src/megax/MegaApp/Vite/src/pages/Login/LoginPage.tsx [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+              label="Username"
+            />
+            <p className="text-red-500 text-xs italic mt-3 text-left">
+              {errors.username?.message}
+            </p>
+          </div>
+          <div className="mb-6">
+            <TextField
+              {...register("password", {
+                required: "Thi is required.",
```
Comment (oclockvn) [inline src/megax/MegaApp/Vite/src/pages/Login/LoginPage.tsx:86]:
```suggestion
                required: "Password is required.",
```

**Review by oclockvn — CHANGES_REQUESTED** [review]

File: src/megax/MegaApp/Vite/src/App.tsx [hunk 4]
```diff
@@ -1,33 +1,22 @@
 import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
+// import reactLogo from './assets/react.svg'
```
[the remark this answers is not in the document]
Comment (tungbinhdinh89) [reply src/megax/MegaApp/Vite/src/App.tsx:2]: yes, oke, i remember.

**Review by oclockvn — APPROVED** [review]

**Pull request merged**